platform/mellanox/mft/Makefile

```diff
@@ -25,7 +25,7 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	popd
 
 	# fix timestamp because we do not actually build tools, only kernel
-	touch $(MFT_NAME)/DEBS/$*
-	mv $(MFT_NAME)/SDEBS/ubuntu-drivers/3.16.0/$(DERIVED_TARGETS) $(MFT_NAME)/DEBS/$* $(DEST)
+	touch $(MFT_NAME)/DEBS/*
+	mv $(MFT_NAME)/SDEBS/ubuntu-drivers/3.16.0/kernel-mft-dkms_4.5.0-$(KVERSION)_all.deb $(MFT_NAME)/DEBS/* $(DEST)
```

Comment:
>kernel [](start = 44, length = 6)

Since the rule is a pattern match one, why you need to specify a special target?

Reply:
kernel-mft-dkms and mft-oem are in DERIVED_TARGETS, but they are placed in different locations, so i specify that kernel-mft-dkms should be moved from $(MFT_NAME)/SDEBS/ubuntu-drivers/3.16.0 and mft, mft-oem packages should be moved from $(MFT_NAME)/DEBS/*